src/legacy/List/context.actions.js

```diff
@@ -31,8 +31,9 @@ function lexicographicalCompare(xs, ys) {
     return _(xs).zipWith(ys, compare).find() || 0;
 }
 
-export async function goToUserEditPage(user) {
+export async function goToUserEditPage(userId) {
     const d2 = await getD2();
+    const user = (await d2.models.users.list({ filter: `id:in:[${userId}]` })).toArray().pop();
```

Comment:
Not sure about this ``.pop`` method. What is the array returning?

Reply:
I'm using `.pop` because the method `d2.models.users.list` returns a list of users and I only need 1. I'm sure I get one because I'm sending a user id. Do you prefer I use `[0]`?